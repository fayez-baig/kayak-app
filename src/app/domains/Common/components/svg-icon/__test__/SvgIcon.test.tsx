import { createRenderer } from "react-test-renderer/shallow";
import SvgIcon from "../SvgIcon";

const renderer = createRenderer();

describe("<SvgIcon />", () => {
  it("should render and match the snapshot", () => {
    renderer.render(<SvgIcon icon="KayakLogoIcon" />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
