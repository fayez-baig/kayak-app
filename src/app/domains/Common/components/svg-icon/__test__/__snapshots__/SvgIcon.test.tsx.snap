// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SvgIcon /> should render and match the snapshot 1`] = `
<ForwardRef(SvgKayakLogo)
  aria-hidden="true"
/>
`;
